lib/dts-lib.robot

```diff
@@ -0,0 +1,130 @@
+*** Settings ***
+Library     ../robot-venv/lib/python3.11/site-packages/robot/libraries/Collections.py
+
+
+*** Keywords ***
+Enable Network Boot
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${nw_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Networking Options
+    Set Option State    ${nw_menu}    Enable network boot    ${TRUE}
+    Save Changes And Reset    2    4
+
+Boot Dasharo Tools Suite
+    [Documentation]    Keyword allows to boot Dasharo Tools Suite. Takes the
+    ...    boot method (from USB or from iPXE) as parameter.
+    [Arguments]    ${dts_booting_method}
```

Comment:
I am not a fan of this argument. If anything, it should be either iPXE or USB. And then, it should automativally detect which USB to look for:
- for PiKVM the name is known
- for QEMU is known
- for others, it should be probably pendrive defined in config, not hardcoded in test

Reply:
[Added](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/1051f8e19ce0e530125e23ea71a2047a6fbc1865#diff-0ca58a364903f744bc9458315dbf5ccc48c37f510ab670103bb67f789341868bR18) support for QEMU detection and config-defined physical USB stick